.github/workflows/verify_source_generated_changes_are_persisted.yml

```diff
@@ -21,6 +21,8 @@ jobs:
 
       - name: "Regenerating package versions"
         run: .\tracer\build.ps1 BuildTracerHome
+        env:
+          NugetPackageDirectory: "${{ github.workspace }}\\packages"
```

Comment:
Do we _need_ this? If so, we probably need to update a _lot_ of workflows...🤔

Reply:
still failing ... do no know why :/
https://github.com/DataDog/dd-trace-dotnet/actions/runs/9571010439/job/26387071993?pr=5614#step:5:10312

Reply:
I just tested and these shouldn't be required now 😃 
```suggestion
```